recordset/recordset.html

```diff
@@ -21,22 +21,43 @@ <h2 id="page-title">{{ vm.tableDisplayName }}</h2>
             <div style="margin-top: 15px;">Loading...</div>
         </div>
 
-        <!-- pagination -->
-        <div style="margin-bottom: 10px;">
-            <span class="dropdown" style="margin-left: 5px">
-                <button class="btn btn-sm btn-default dropdown-toggle" type="button" data-toggle="dropdown">Items per page
-                    <span class="caret"></span></button>
-                <ul class="dropdown-menu">
-                    <li ng-if="pageLimits.indexOf(pageInfo.pageLimit) === -1"><a id="custom-page-size" href ng-click="pageLimit(pageInfo.pageLimit)">
-                        <span class="glyphicon glyphicon-ok"></span>
-                        {{ pageInfo.pageLimit }} (Custom)</a>
-                    </li>
-                    <li ng-repeat="limit in pageLimits"><a href ng-click="pageLimit(limit)">
-                        <span class="glyphicon" ng-class="pageInfo.pageLimit === limit ? 'glyphicon-ok' : 'glyphicon-invisible'"></span>
-                        {{ limit }}</a>
-                    </li>
-                </ul>
-            </span>
+        <div class="row" style="margin-bottom: 10px;">
+            <div class="col-lg-4">
+                <!-- search box -->
+                <div class="input-group">
+                    <input type="text"
+                           id="search-input"
+                           ng-model="vm.search"
+                           class="form-control"
+                           enter-key="search(vm.search)"
```

Comment:
More a personal opinion but I would think that naming your directive `onEnter` would be more intuitive than `enterKey`. That way it can read like "on Enter run this `search` function".